booq/deployment.yaml

```
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: booq
+  labels:
+    app: booq
+
+spec:
+  selector:
+    matchLabels:
+      app: booq
+  template:
+    metadata:
+      labels:
+        app: booq
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: las211.tokyotech.org
+
+      containers:
+
+        - name: booq-frontend
+          image: ghcr.io/traptitech/booq-ui:latest@sha256:d3e9da936f0300842eb9ea30dbc105bb4d5e3e0c8a660bfeb303f9de38041dab
```

latestではなくバージョン指定にして欲しいです！[参考](https://github.com/traPtitech/booQ-UI/pkgs/container/booq-ui)

```suggestion
          image: ghcr.io/traptitech/booq-ui:2.1.3
```